bitnami/contour/README.md

```diff
@@ -55,7 +55,7 @@ The following tables lists the configurable parameters of the contour chart and
 | `rbac.create`                                      | create the RBAC roles for API accessibility                                                            | `true`                                                  |
```

Comment:
Could we add a small note at the end in the `Upgrading` section telling all the big changes regarding contour and also adding something like this for the bitnami/common addition?

```
## Upgrading

### To 2.0.0
...
<most important changes here>
...

This version also introduces `bitnami/common`, a [library chart](https://helm.sh/docs/topics/library_charts/#helm) as a dependency. More documentation about this new utility could be found [here](https://github.com/bitnami/charts/tree/master/bitnami/common#bitnami-common-library-chart). Please, make sure that you have updated the chart dependencies before executing any upgrade.
```

Reply:
Sure. Already pushed it.